product_docs/docs/pgd/5/node_management/connections_dsns_and_ssl.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Connection DSNs and SSL (TLS)
+---
+
+The DSN of a node is simply a `libpq` connection string, since nodes connect
```

Comment:
```suggestion
The DSN of a node is a [`libpq`]( https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNECT-SSLMODE) connection string, since nodes connect
```